src/cpp/transport/serial/TermiosAgentLinux.cpp

```diff
@@ -53,17 +53,14 @@ bool TermiosAgent::init()
 
     // Check if serial port exist
     int serial_exist = access(dev_.c_str(), F_OK );
-    if( serial_exist != 0 ) {
+    while(serial_exist != 0)
```

Comment:
Can we use a a `do {} while()` approach?

Is possible to `sleep_for` less (maybe 10 ms) but print the message every 1s? I think that is important to detect when the serial port is available because the Client on the other side of the cable can timeout if the agent takes too much in provide an answer.

Reply:
Updated:
https://github.com/eProsima/Micro-XRCE-DDS-Agent/blob/77c8ca785df1a04a5442ec918e89fde42112b14d/src/cpp/transport/serial/TermiosAgentLinux.cpp#L56-L85